buttons/buttons.css

```diff
@@ -2597,6 +2597,37 @@ a:focus-visible {
 }
   /* radzhiv's btn 2 end */
 
+/* abhishek's btn 1 start */
+  .abhishek0-btn-1 {
+    background-color: initial;
```

Comment:
pls change your classes names to as i said it should be your github-username not name so githubusername-btn-1

Reply:
also pls be creative this is just simple change